beetsplug/lastgenre/genres-tree.yaml

```diff
@@ -156,6 +156,31 @@
     - timba
     - twoubadou
     - zouk
+- classic:
```

Comment:
Wouldn't `classical` be a better "super genre"? `classic` sounds a bit odd to me.